### PR TITLE
Update teamcenter-extension.md

### PR DESCRIPTION
**Pull request by wanchoorohan**
1. Updated documentation for V2
2. In some sections, I provided details for both V1 and V2. Moving forward I wonder if it makes sense to just provide V2 details?

### CODE REVIEW

File: content/en/docs/appstore/use-content/modules/teamcenter-extension.md [hunk 1]
```diff
@@ -23,55 +23,68 @@ The extension offers the following integration options:
...
-### 1.3 Pre-requisites {#prerequisities}
+### 1.3 Prerequisites {#prerequisites}
 
-You must use Teamcenter Extension with a Studio Pro version between 10.6.5 to 10.8. Mendix recommends using Studio Pro version [10.6.7](/releasenotes/studio-pro/10.6/#1067), as it is the medium-term support (MTS) version.
+The compatibility between Teamcenter Extension and Studio Pro is as follows:
 
-### 1.4 Dependencies 
+* Teamcenter Extension V 1.0.0 is compatible with Studio Pro versions between 10.6.5 and 10.7.
+
+* Teamcenter Extension V 2.0.0 is compatible with Studio Pro version 10.12 or higher.
```
Comment (wanchoorohan) [inline content/en/docs/appstore/use-content/modules/teamcenter-extension.md]: Please change this to 10.12.1

File: content/en/docs/appstore/use-content/modules/teamcenter-extension.md [hunk 2]
```diff
@@ -102,50 +115,89 @@ This feature is designed specifically for generating microflows and domain model
...
+For details on the version dependencies between Studio Pro and Teamcenter Extension, see the [Prerequisites](#prerequisites) section.
+{{% /alert %}}
+
+The landing page has with three tabs: **Menu**, **History**, and **Settings**.
+
+#### 3.2.1 Menu Tab
+
+The **Menu** tab displays use cases or actions you can create artifacts for using Teamcenter Extension.
+
+{{< figure src="/attachments/appstore/use-content/modules/teamcenter-extension/menu-tab.png" >}}
```
Comment (wanchoorohan) [inline content/en/docs/appstore/use-content/modules/teamcenter-extension.md:132]: Please replace this image with this one - 

screenshot_V2

File: content/en/docs/appstore/use-content/modules/teamcenter-extension.md [hunk 3]
```diff
@@ -102,50 +115,89 @@ This feature is designed specifically for generating microflows and domain model
...
+
+#### 3.2.2 History Tab
+
+The **History** tab displays the history of all actions (also referred as integrations) carried out in Teamcenter Extension.
+
+{{< figure src="/attachments/appstore/use-content/modules/teamcenter-extension/history-tab.png" >}}
```
Comment (wanchoorohan) [inline content/en/docs/appstore/use-content/modules/teamcenter-extension.md:138]: Replace this image with this one 
screenshot_V2_2

File: content/en/docs/appstore/use-content/modules/teamcenter-extension.md [hunk 4]
```diff
@@ -102,50 +115,89 @@ This feature is designed specifically for generating microflows and domain model
...
+* Moving microflows
+* Renaming microflows
+
+If you select an item in the **Action** list, Teamcenter Extension performs a validation check to see if the Teamcenter objects, entities, and microflows still exist and are valid. The results are reported in the **Validation** section.
 
-The **Menu** tab displays use cases or actions you can create artifacts for using the extension.
+#### 3.2.3 Settings Tab
 
-{{< figure src="/attachments/appstore/use-content/modules/teamcenter-extension/settings-tab.png" max-width=50% >}}
+The **Settings** tab displays the Teamcenter configuration page.
```
Comment (wanchoorohan) [inline content/en/docs/appstore/use-content/modules/teamcenter-extension.md:164]: Replace this image with this one 
screenshot_V2_3

**Review by wanchoorohan — COMMENTED** [review]
I added few comments. Thanks

**Comment by Luyao-Zhang-1** [comment]
> I added few comments. Thanks

Thank you for your comments. I implemented your suggestions.

File: content/en/docs/appstore/use-content/modules/teamcenter-extension.md [hunk 5]
```diff
@@ -23,55 +23,68 @@ The extension offers the following integration options:
...
+
+* [Teamcenter Connector](https://marketplace.mendix.com/link/component/111627): needed for all versions of Teamcenter Extension
+
+  The table below details the compatibility between Teamcenter Extension, Teamcenter Connector, and Studio Pro.
 
-* [Community Commons](https://marketplace.mendix.com/link/component/170)
+| Teamcenter Extension Version | Teamcenter Connector | Studio Pro Version |
+| ------------- | ------------- | ------------- |
+| 1.0.0  | 3.6.1, 3.6.0, 3.5.0  | 10.6.5 thru 10.7  |
+| 2.0.0  | 2406  | 10.12.1 and above  |
```
Comment (wanchoorohan) [inline content/en/docs/appstore/use-content/modules/teamcenter-extension.md:54]: We could put a line here stating - "If you are using Teamcenter Extension 1.0.0 along with Teamcenter Connector 3.6.1 or below and want to upgrade to Teamcenter Extension 2.0.0 and Teamcenter Connector 2406, please refer to section xxxx"

**Pull request merged**